demisto_sdk/commands/common/hook_validations/description.py

```diff
@@ -133,15 +133,23 @@ def is_duplicate_description(self):
     @error_codes('DS106')
     def is_valid_description_name(self):
         """Check if the description name is valid"""
+        integrations_folder = ''
+        description_file_base_name = ''
         description_path = glob.glob(os.path.join(os.path.dirname(self.file_path), '*_description.md'))
         md_paths = glob.glob(os.path.join(os.path.dirname(self.file_path), '*.md'))
+        for path in description_path:
+            description_file = os.path.basename(path)
+            integrations_folder = os.path.basename(os.path.dirname(path))
+
+            # drop file extension
+            description_file_base_name = description_file.rsplit('_', 1)[0]
```

Comment:
Don't really see a reason for the for loop here. Also, let's put this logic before line 138.
```suggestion

        description_file = os.path.basename(path)
        integrations_folder = os.path.basename(os.path.dirname(path))

        # drop file extension
        description_file_base_name = description_file.rsplit('_', 1)[0]
```

Reply:
The code here is written like this:
`description_path = glob.glob(os.path.join(os.path.dirname(self.file_path), '*_description.md'))`
The description path returns a list, that's why the for loop is here.
I didn't want to refactor this at the moment